src/test/ui/nll/issue-47388.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0594]: cannot assign to data in a `&` reference
+  --> $DIR/issue-47388.rs:18:5
+   |
+LL |     let fancy_ref = &(&mut fancy);
+   |                     ------------- help: consider changing this to be a mutable reference: `&mut`
```

Comment:
I believe either the span or the suggestion text might be incorrect. It should either suggest `&mut (&mut fancy)` for this span, or use a span only selecting the `&`.

Reply:
Filed #49859